docs/content/preview/explore/_index.md

```diff
@@ -12,151 +12,183 @@ aliases:
 
-  <div class="col-12 col-md-6 col-lg-12 col-xl-6">
-    <a class="section-link icon-offset" href="query-1-performance/">
-      <div class="head">
-        <img class="icon" src="/images/section_icons/explore/fault_tolerance.png" aria-hidden="true" />
-        <div class="title">Query tuning</div>
-      </div>
-      <div class="body">
-        Optimize query performance.
-      </div>
+<ul class="nav nav-tabs-alt nav-tabs-yb custom-tabs">
+  <li >
+    <a href="#cloud" class="nav-link active" id="cloud-tab" data-toggle="tab"
+       role="tab" aria-controls="cloud" aria-selected="true">
+      <i class="fas fa-cloud" aria-hidden="true"></i>
+      Use a cloud cluster
     </a>
-  </div>
-
-  <div class="col-12 col-md-6 col-lg-12 col-xl-6">
-    <a class="section-link icon-offset" href="change-data-capture/">
-      <div class="head">
-        <img class="icon" src="/images/section_icons/explore/monitoring.png" aria-hidden="true" />
-        <div class="title">Change data capture</div>
-      </div>
-      <div class="body">
-        Stream database changes to external processes, applications, or other databases.
-      </div>
+  </li>
+  <li>
+    <a href="#local" class="nav-link" id="local-tab" data-toggle="tab"
+       role="tab" aria-controls="local" aria-selected="false">
+      <i class="icon-shell" aria-hidden="true"></i>
+      Use a local cluster
     </a>
-  </div>
+  </li>
+</ul>
 
-  <div class="col-12 col-md-6 col-lg-12 col-xl-6">
-    <a class="section-link icon-offset" href="security/security/">
-      <div class="head">
-        <img class="icon" src="/images/section_icons/explore/secure.png" aria-hidden="true" />
-        <div class="title">Security</div>
-      </div>
-      <div class="body">
-        Learn about strategies for securing YugabyteDB.
-      </div>
-    </a>
-  </div>
+<div class="tab-content">
+  <div id="cloud" class="tab-pane fade show active" role="tabpanel" aria-labelledby="cloud-tab">
 
-  <div class="col-12 col-md-6 col-lg-12 col-xl-6">
-    <a class="section-link icon-offset" href="observability/">
-      <div class="head">
-        <img class="icon" src="/images/section_icons/explore/monitoring.png" aria-hidden="true" />
-        <div class="title">Observability</div>
-      </div>
-      <div class="body">
-        Observe metrics at every layer of the database using Prometheus.
-      </div>
-    </a>
-  </div>
+To run the Explore examples in YugabyteDB Managed, create a single- or multi-node cluster as follows:
+
+- Single node
+
+  Examples requiring a single-node cluster can be run using the free [Sandbox](../yugabyte-cloud/cloud-basics/create-clusters/create-clusters-free/) cluster.
+
+  If you haven't already created your sandbox cluster, log in to YugabyteDB Managed, on the **Clusters** page click **Add Cluster**, and follow the instructions in the **Create Cluster** wizard.
+
+  Save your cluster credentials in a convenient location, you will use them to connect to your cluster.
```

Comment:
Suggestion:
Save your cluster credentials in a convenient location. You will use them to connect to your cluster.